burrow/src/daemon/net/apple.rs

```diff
@@ -16,9 +19,12 @@ pub extern "C" fn start_srv(){
     let rt = Runtime::new().unwrap();
     rt.block_on(async {
         loop {
-            if let Ok(_) = DaemonClient::new().await{
-                break
+            match DaemonClient::new().await {
+                Ok(_) => break,
+                Err(_e) => {
+                    // error!("Error when connecting to daemon: {}", e)
+                }
             }
```

Comment:
Can we log both success and failure here?
```
match DaemonClient::new().await {
    Ok(..) => info!("Server successfully started")
    Err(e) => error!("Could not connect to server: {}", e)
}
```
and, does it need to be in a loop? I worry about it looping a lot in the error case

Reply:
Updated to log both!

The loop is used currently as a way to detect if the daemon server has started. 
I guess we can try setting a timeout/threshold to the amount of attempts to make?
A better way to do this would be to use something like `tokio::sync::Notify` to notify the blocking thread when the server has finished initializing.